src/js/utils/date.util.js

```diff
@@ -1,87 +1,239 @@
-import moment from 'moment'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
+import duration from 'dayjs/plugin/duration'
+import customParseFormat from 'dayjs/plugin/customParseFormat'
+import isSameOrAfter from 'dayjs/plugin/isSameOrAfter'
+import isBetween from 'dayjs/plugin/isBetween'
 
-const INPUT_DATE_FORMAT = 'DD/MM/YYYY'
+import { LOCALES } from '@/js/const/date-locales.const'
+
+dayjs.extend(utc)
+dayjs.extend(duration)
+dayjs.extend(customParseFormat)
+dayjs.extend(isSameOrAfter)
+dayjs.extend(isBetween)
 
 export class DateUtil {
-  static get ISOFormat () {
-    return 'YYYY-MM-DDT00:00:00+00:00'
+  static _dayjs (date, format) {
+    return format ? dayjs(date, format) : dayjs(date)
```

Comment:
It needs special attention, because dayjs have three modes - local timezone, setted timezone, utc
Momentjs can only work in two modes: timezone and utc
dayjs() works always in local timezone, even if you set another timezone by dayjs.tz.setDefault('Your timezone')
dayjs.tz() works in setted timezone or default local timezone
dayjs.utc() works, respectively, in utc timezone